qiskit_experiments/test/Fine Calibration Tutorial.rst

```diff
@@ -0,0 +1,154 @@
+containing such repeated gates within a circuit, can be run efficiently in the "IQPulseBackend"
+
+Let's start with importing some necessary modules and build our pulse simulator with
+"SingleTransmonTestBackend" which is inherited from "IQPulseBackend"
+
+.. jupyter-execute:: 
+
+    import numpy as np
+    from qiskit.pulse import InstructionScheduleMap
+    import qiskit.pulse as pulse
+    from qiskit_experiments.library import FineXAmplitude, FineSXAmplitude
+    from qiskit_experiments.test.iq_pulse_backend import SingleTransmonTestBackend
+
+.. jupyter-execute::
+
+    backend = SingleTransmonTestBackend()
+    qubit = 0
+-----------------------------------------------------
+Fine X gate Amplitude Calibration
+-----------------------------------------------------
+We will run the fine calibration experiments with our own pulse schedules. 
+To do this we create an instruction to schedule map which we populate with 
+the schedules we wish to work with. This instruction schedule map is then 
+given to the transpile options of the calibration experiments so that 
+the Qiskit transpiler can attach the pulse schedules to the gates in the experiments. 
+We will base all our pulses on the default X pulse of "SingleTransmonTestBackend".
+
+.. jupyter-execute::
+    x_pulse = backend.defaults().instruction_schedule_map.get('x', (qubit,)).instructions[0][1].pulse
+    d0, inst_map = pulse.DriveChannel(qubit), InstructionScheduleMap()
+
+    for name, pulse_ in [("x", x_pulse)]:
+        with pulse.build(name=name) as sched:
+            pulse.play(pulse_, d0)
+
+        inst_map.add(name, (qubit,), sched)
+
+We now take the ideal x pulse amplitude reported by the backend and 
+add/subtract a 2% over/underrotation to it by scaling the ideal amplitude and see 
+if the experiment can detect this over/underrotation. We replace the default X pulse 
+in the instruction schedule map with this over/underrotated pulse.
+
+.. jupyter-execute::
+    ideal_amp = x_pulse.amp
+    over_amp = ideal_amp*1.02
+    under_amp = ideal_amp*0.98
+    print(f"The reported amplitude of the X pulse is {ideal_amp:.4f} which we set as ideal_amp. we use {over_amp:.4f} amplitude for overroation pulse and {under_amp:.4f} for underrotation pulse")
+
+-----------------------------------------------------------------------------------
+Detecting an over & under rotated pulse
+-----------------------------------------------------------------------------------
+
+.. jupyter-execute::
+    # build the over rotated pulse and add it to the instruction schedule map
+    with pulse.build(backend=backend, name="x") as x_over:
+        pulse.play(pulse.Drag(x_pulse.duration, over_amp, x_pulse.sigma, x_pulse.beta), d0)
+    inst_map.add("x", (qubit,), x_over)
+
+    # do the experiment
+    overamp_cal = FineXAmplitude(qubit, backend=backend)
+    overamp_cal.set_traspile_options(inst_map=inst_map)
+    exp_data_over = overamp_cal.run(backend)
+    exp_data_over.figure(0)
```

Comment:
Can you make a cell that prints out one of the circuits? Perhaps one showing 3 X gates? Also make the comment that, for the calibration of the X gate we have a square-root of X gate before the X gates. This is there to move the ideal population to the equator of the Bloch sphere where the population change is most sensitive to over/under rotations.

Reply:
I'm confused with the purpose of square root X.
1. you said the rason for SX is the sensitivity of equator population change.
Q1. Isnt it right that the change in population independent of qubit intial state? Whether the qubit was in it's ground state or superposition state, isnt the over/under rotation amplitude affect the qubit in the same way?
2. In the qiskit-experiment tutorial, they say the reason for SX is to be able to distinguish between over-rotated and under-rotated pulses.
Q2. I think we can distinguish between the over and under rotated pulse even when qubit starts from the ground state. Isnt it??